src/server/com/yetanalytics/datasim/server.clj

```diff
@@ -195,32 +195,7 @@
                        {:status 401
                         :body   "No Authorization"}))))})
 
-(def download-url
-  {:name  :datasim.route/download-url
-   :enter (fn [context]
-            (assoc context
-                   :response
-                   (try
-                     {:status 200
-                      :body   (-> context
-                                  :request
-                                  :query-params
-                                  :url
-                                  codec/url-decode
-                                  client/get
```

Comment:
Are these two functions still used elsewhere or are they now unused?

Reply:
I saw the removed imports, but I was thinking whether these functions should be removed entirely.

Reply:
client, as mentioned, is used in xapi-client. ring codec as far as i can tell is riding in on another dep already (maybe pedestal).